sys/include/net/ng_netreg.h

```diff
@@ -75,6 +75,9 @@ void ng_netreg_init(void);
  * @param[in] entry     An entry you want to add to the registry with
  *                      ng_netreg_entry_t::pid and ng_netreg_entry_t::demux_ctx set.
  *
+ * @warning Call ng_netreg_unregister() *before* you leave the context you
+ *          allocated @p entry in. Otherwise it might get overwritten.
+ *
```

Comment:
Sorry if this is a stupid question but shouldn't this warning be part of the documentation for `ng_netreg_unregister()`?